lib/commands/search.js

```diff
@@ -0,0 +1,27 @@
+(function () {
+    "use strict";
+
+    var Utils = require('./utils'),
+        Queries = require('./queries'),
+        Q = require('q');
+
+    class SearchCommand {
+        constructor(db) {
+            this.db = db;
+            this.prefix = '.search';
+            this.usage = '.search';
+            this.description = 'Search all columns in all tables for a value (takes a while)';
+        }
+
+        run(messages, writer, args) {
+            if (!Utils.validateArgs(args, 1)) {
+                return Q.reject('No search value supplied');
+            }
+
```

Comment:
maybe we can write a message to screen before the command runs, something like 'Searching...'
please make sure that the message is only written to console and does not become part of file output